tsconfig.json

```diff
@@ -15,7 +15,11 @@
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "baseUrl": "./",
+    "paths": {
+      "@bookoid/components": ["./components"]
```

Comment:
Why do we want to alias imports? Personally, I'm not a fan of this because it encourages wild directory structures since we can just hide behind an alias. I'd prefer we keep the directory structure clean so our components always import from `./` or `../` at worst instead of having this.